querydsl-codegen/src/main/java/com/querydsl/codegen/EntitySerializer.java

```diff
@@ -64,18 +64,25 @@ public EntitySerializer(TypeMappings mappings, @Named("keywords") Collection<Str
         this.keywords = keywords;
     }
 
+    private boolean superTypeHasEntityFields(EntityType model) {
+        Supertype superType = model.getSuperType();
+        return null != superType && null != superType.getEntityType() && superType.getEntityType().hasEntityFields();
```

Comment:
For consistency's sake, could you please add a newline right before the second `&&`?